src/main/scala/com/databricks/labs/overwatch/pipeline/Silver.scala

```diff
@@ -368,6 +372,21 @@ class Silver(_workspace: Workspace, _database: Database, _config: Config)
       )
   }
 
+  lazy private[overwatch] val warehouseSpecModule = Module(2021, "Silver_WarehouseSpec", this, Array(1004))
+  lazy private val appendWarehouseSpecProcess: () => ETLDefinition = {
+    () =>
+    ETLDefinition(
+      BronzeTargets.auditLogsTarget.asIncrementalDF(warehouseSpecModule, BronzeTargets.auditLogsTarget.incrementalColumns),
+      Seq(
+        buildWarehouseSpec(
```

Comment:
I think you're going to need to send in full silver warehouse target itself so you can fill forward appropriately.